src/serializer/ResidualHeapSerializer.js

```diff
@@ -2202,12 +2242,12 @@ export class ResidualHeapSerializer {
     this.rewrittenAdditionalFunctions.set(additionalFunctionValue, body);
   }
 
-  prepareAdditionalFunctionValues() {
+  prepareOptimizedFunctionValues() {
```

Comment:
Nit: the renaming change makes it harder to see what actually changed in the logic vs what was renamed. In the future it might be better to separate the renames from the behavior differences.

Reply:
I thought I'd rename it on this PR as this PR was quite tiny in size and it is somewhat related.

Reply:
I understand—I'm just saying even splitting this up in two commits would make it easier for review to a person unfamiliar with the code (like me)

Reply:
Will do.